Sources/AccountSDKIOSWeb/Simplified Login/SimplifiedLoginViewController.swift

```diff
@@ -1,15 +1,44 @@
 import SwiftUI
 import UIKit
 
-public struct SimplifiedLoginViewModel {
+public struct SimplifiedLoginUIFactory {
+    
+    public static func buildViewController(client: Client,
+                                           env: ClientConfiguration.Environment,
+                                           clientStartLoginSession: @escaping () -> Void ) -> UIViewController {
```

Comment:
I am thinking that brands should pass in how they want to start a login session, similar as in line 275.
The reason I am thinking about doing this way, is so brands easier can call `Client.getLoginSession` with their own configuration. 

A downside is that this approach does not force the use of `client.getLoginSession` on user button click `Switch Account`. A different approach would be to extend the signature of `SimplifiedLoginUIFactory.buildViewController` with the parameters needed to call `client.getLoginSession`. Both for iOS >13 and below.

@zamorastp what do you think?

Reply:
Let me think about it.

Reply:
Definitely, brands should have a possibility to choose how they want to start a login session. If we do not force the use of client.getLoginSession, probably they are not going to do that (what are the consequences ? they can just do whatever they want?). Are there any other options which they can use? 
The solution should be simple, but also foolproof and possibly not so hermetic. Does the second solution force to use getLoginSession and also add a possibility to choose how to start login session? If yes, lets do that

Reply:
I added two separate build `buildViewController` functions. 
One for pre iOS 13 and another for above.